### ISSUE TITLE
Unable to find target for this triple (no targets are registered)

### ISSUE THREAD

**Issue by Frithurik** [llvm/llvm-project#72766]
![immagine](https://github.com/llvm/llvm-project/assets/138444950/a0336e45-8db6-4cb9-9dab-5fa280174a16)

I got this error while compilng Kaleidoscope example. I checked TargetRegistry::targets() and it's empty. I'm compiling with Visual Studio 2022 with x86_64-pc-windows-msvc target triple. I installed LLVM with VCPKG.

**Comment by AntoineGagnon1** [comment]
@Frithurik I am having the same issue right now, did you find a fix ?

**Comment by Frithurik** [comment]
No, I didn't, but I installed LLVM on another PC and on that it works.

**Comment by AntoineGagnon1** [comment]
I fixed it by manually linking those dll :
```
LLVMX86Info.lib
LLVMX86TargetMCA.lib
LLVMX86Disassembler.lib
LLVMX86Desc.lib
LLVMX86CodeGen.lib
```

**Comment by Frithurik** [comment]
Ok, I will try.

**Comment by volkaris** [comment]
> Ok, I will try.

did you fixed that? i have same problem

**Comment by volkaris** [comment]
> I fixed it by manually linking these dlls :
> 
> ```
> LLVMX86Info.lib
> LLVMX86TargetMCA.lib
> LLVMX86Disassembler.lib
> LLVMX86Desc.lib
> LLVMX86CodeGen.lib
> ```

Hi?can you give list of lib that you linking? i got same error. and im linking all dlls.
